client/src/js/services/util.js

```diff
@@ -13,11 +13,40 @@ UtilService.$inject = ['moment' ];
  */
 function UtilService(moment) {
   var service = this;
-
+  
   service.unwrapHttpResponse = function unwrapHttpResponse(response) {
     return response.data;
   };
 
+  service.formatDate = function(date, format) {
+    var f = format || 'DD/MM/YYYY HH:mm:ss';
+    if(date){
```

Comment:
Could you make sure this respects eslint?  There should be a space between the `if` and all brackets.

````js
if (date) {
// etc ...
```